src/main/java/com/kuliginstepan/dadata/client/domain/AdditionalProps.java

```diff
@@ -0,0 +1,25 @@
+package com.kuliginstepan.dadata.client.domain;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AdditionalProps {
```

Comment:
Add some javadoc about `AdditionalProps`. For example, `AdditionalProps` is a map, which contains unmapped properties from dadata API